app/uk/gov/hmrc/icedsubscriptionfrontend/views/SuccessfullyEnrolledPage.scala.html

```diff
@@ -34,14 +34,16 @@
 
     @paragraph("successfullyEnrolled.para.1")
 
-    @paragraph("successfullyEnrolled.coms")
+    @paragraph("successfullyEnrolled.para.1")
+
+    @hmrcPageHeading(PageHeading(text = messages("successfullyEnrolled.heading")))
 
-    @paragraph("successfullyEnrolled.incorporates")
-    @bulletList(Seq(Html(messages("successfullyEnrolled.incorporates.1")), Html(messages("successfullyEnrolled.incorporates.2"))))
+    @paragraph("successfullyEnrolled.para.1")
+
+    @paragraph("successfullyEnrolled.para.1")
```

Comment:
This should probably be in the paragraph with line above but with a page break in between the two lines